src/components/Category.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import CategoryProgress from './CategoryProgress'
+import SubCategory, { subCategoryTotal } from './SubCategory'
+
+const StyledCategory = styled.div`
+  width: 80%;
+  .CategoryWrapper {
+    display: flex;
+    align-items: center;
+  }
+  .CategoryHeader {
+    display: flex;
+    align-items: center;
+  }
+  .CategoryCircle {
+    margin: 10px;
+    background: #46237a;
+    border-radius: 50%;
+    width: 50px;
+    height: 50px;
+  }
+  .CategoryTitle {
+    color: #4a4a4a;
+    font-size: 22px;
+    font-weight: 500;
+  }
+  .CategoryDescription {
+    color: #9b9b9b;
+    font-size: 12px;
+  }
+  .CategoryRightChild {
+    margin-left: auto;
+  }
+`
+
+const data = {
+  id: 'entertainment',
+  title: 'Entertainment',
+  description: 'Pleasure such as dining out, concerts, etc',
+  budget: 500,
+  subCategories: [
+    { title: 'Books/Movies/Music', total: 20 },
+    { title: 'Dining Out', total: 125 },
+    { title: 'Concerts/Sports Events', total: 50 },
+    { title: 'Vacation/Experiences', total: 100 },
+    { title: 'Other', total: 25 },
+  ],
+}
+
+const categoryTotal = data.subCategories.reduce(
+  (nextTotal, subCategoryBudget) => nextTotal + subCategoryBudget.total,
+  0
+)
+
+export default function Category() {
+  return (
+    <StyledCategory>
+      <div className="CategoryWrapper">
+        <div className="CategoryLeftChild">
```

Comment:
**OPTIONAL** in the future, we probably want to use more generic class names like `ShiftLeft` or better yet, abstract the group into it's own `styled-component` file so they can be reused in multiple places, but it's totally fine for now